src/test/java/api/loans/DeclareLostAPITests.java

```diff
@@ -567,6 +722,22 @@ private void assertNoFeeAssignedForLoan(UUID loan) {
     assertThat(getAccountsForLoan(loan), hasSize(0));
   }
 
+  private List<JsonObject> getOpenAccounts(List<JsonObject> accounts) {
+    return accounts.stream().filter(account -> isOpenAccount(account)).collect(Collectors.toList());
```

Comment:
A method reference can be used instead of a lambda, something like
```suggestion
    return accounts.stream().filter(this::isOpen).collect(Collectors.toList());
```